documentation/specs/ToggleCard.md

```diff
@@ -0,0 +1,209 @@
+
+### Example Usage
+
+_Controlled:_
+
+```tsx
+import { ToggleCard } from "@easypost/easy-ui/ToggleCard";
+import { Text } from "@easypost/easy-ui/Text";
+
+function Component() {
+  const [isSelected, setIsSelected] = useState(false);
+  return (
+    <ToggleCard>
+      <ToggleCard.Header
+        isSelected={isSelected}
+        onChange={(isSelected) => setIsSelected(isSelected)}
+        aria-labelledby="some id"
+      >
+        <Text id="some id" variant="subtitle1" color="primary.900">
+          Header
+        </Text>
+      </ToggleCard.Header>
+      <ToggleCard.Body>Content</ToggleCard.Body>
+    </ToggleCard>
+  );
+}
+```
+
+_Flipped alignment:_
+
+```tsx
+import { ToggleCard } from "@easypost/easy-ui/ToggleCard";
+import { Icon } from "../Icon";
+
+
+function Component() {
+  const PoweredByEasyPostLogo = () => <Image src="./logo.png" />;
+  return (
+    <ToggleCard>
+      <ToggleCard.Header aria-label="carrier activation" isAligmentFlipped>
+        <Icon size="sm" symbol={PoweredByEasyPostLogo}>
+      </ToggleCard.Header>
+      <ToggleCard.Body>
+        <button>
+          <Image src="./carrier.png" />;
+        </button>
+      </ToggleCard.Body>
+    </ToggleCard>
+  );
+}
+```
+
+_Disabled:_
+
+```tsx
+import { ToggleCard } from "@easypost/easy-ui/ToggleCard";
+import { Text } from "@easypost/easy-ui/Text";
+
+function Component() {
+  return (
+    <ToggleCard>
+      <ToggleCard.Header isDisabled aria-labelledby="some id">
+        <Text id="some id" variant="subtitle1" color="primary.900">
+          Header
+        </Text>
+      </ToggleCard.Header>
+      <ToggleCard.Body>Content</ToggleCard.Body>
+    </ToggleCard>
+  );
+}
+```
+
+_Read only:_
+
+```tsx
+import { ToggleCard } from "@easypost/easy-ui/ToggleCard";
+import { Text } from "@easypost/easy-ui/Text";
+
+function Component() {
+  return (
+    <ToggleCard>
+      <ToggleCard.Header isSelected isReadOnly aria-labelledby="some id">
+        <Text id="some id" variant="subtitle1" color="primary.900">
+          Header
+        </Text>
+      </ToggleCard.Header>
+      <ToggleCard.Body>Content</ToggleCard.Body>
+    </ToggleCard>
+  );
+}
+```
+
+### Anatomy
+
+```tsx
+import { Toggle } from "../Toggle";
+import { Card } from "../Card";
+import { HorizontalGrid } from "../HorizontalGrid";
+
+export function ToggleCard(props: ToggleCardProps) {
+  const { children } = props;
+
+  return (
+      <Card.Container>{children}</Card.Container>
+  );
+}
+
+function ToggleCardHeader(props: ToggleCardHeaderProps) {
+  const { children, isAligmentFlipped = false, ...toggleProps } = props;
+
+  return (
+    <HorizontalStack>
+      <Card.Area>
+        {isAlignmentFlipped ? (
+          <>
+            <Toggle {...toggleProps} />
+            {children}
+          </>
+        ) : (
+          {children}
+          <Toggle {...toggleProps} />
+        )}
+      </Card.Area>
+    </HorizontalStack>
+  );
+}
+
+function ToggleCardBody(props: ToggleCardBodyProps) {
```

Comment:
anatomy may change to account for additional vertical padding in the card's body